tests/testthat/_snaps/compat-types-check.md

```diff
@@ -88,6 +88,59 @@
       Error in `checker()`:
       ! `foo` must be a single string, not a number.
 
+# `check_name()` checks
+
+    Code
+      err(checker("", check_name))
+    Output
+      <error/rlang_error>
+      Error in `checker()`:
+      ! `foo` must be a single string, not `""`.
```

Comment:
Would be nice if this could be `must be a valid variable name`

Reply:
I was worried that referring to "name" might be a bit too restricting. "variable name" feels especially too specific to me, but we could try `must be a valid name"`?

Reply:
"Valid name" feels slightly too unspecified to me, but I can't think of anything better.